docs/components/empty-state.md

```diff
@@ -115,6 +114,52 @@ export default {
 </script>
 ```
 
+### is-error
+- `is-error`
+
+A flag denoting whether or not the message is an error message. If so, a warning icon is displayed above the title slot.
+
+<template>
+  <KEmptyState is-error="true">
```

Comment:
The is-error examples need to have `cta-is-hidden` so they don't render an empty cta button